lib/whitehall_importer/create_file_attachment_revision.rb

```diff
@@ -18,18 +19,19 @@ def call
       check_file_requirements(decorated_file)
 
       blob_revision = create_blob_revision(decorated_file)
-      FileAttachment::Revision.create!(
+      revision = FileAttachment::Revision.create!(
         blob_revision: blob_revision,
         file_attachment: FileAttachment.create!,
         metadata_revision: FileAttachment::MetadataRevision.create!(
           title: whitehall_file_attachment["title"],
         ),
       )
+      record_assets(revision)
```

Comment:
Minor but It seems a little strange that this record_assets returns a revision. 

You could do `revision.tap { |r| record_assets(r) }` instead? or just a:
````
record_assets(revision)
revision
```

You might find it's more intuitive to call this method with a tap maybe? revision